### PR TITLE
Fix entites escape

### PR DESCRIPTION
**Pull request by pinnisi**

### CODE REVIEW

File: lib/xml/utils/entities.dart [hunk 1]
```diff
@@ -339,9 +339,17 @@ String _encodeXmlText(String input) {
   return input.replaceAllMapped(_textPattern, _textReplace);
 }
 
-final Pattern _textPattern = new RegExp(r'[&<]');
+final Pattern _textPattern = new RegExp('([&<]|\]\]>)');
```
Comment (renggli) [inline lib/xml/utils/entities.dart]: I don't think the parens around the regexp are needed.

**Comment by pinnisi** [comment]
Let me know if it's ok now

**Comment by renggli** [comment]
Merged. Thanks a lot for your fix. I will create a new release on pub in a couple of days, when I have access to a full machine again.

**Comment by renggli** [comment]
Published  2.6.0 to https://pub.dartlang.org/packages/xml.

**Pull request merged**